121.best-time-to-buy-and-sell-stock.py

```diff
@@ -0,0 +1,25 @@
+#
+# @lc app=leetcode id=121 lang=python3
+#
+# [121] Best Time to Buy and Sell Stock
+#
+
+# @lc code=start
+class Solution:
+    def maxProfit(self, prices: List[int]) -> int:
+        """
+        time: O(len(prices))
+        space: O(len(prices))
+        """
+        if not prices:
+            return ""
+
+        min_price = prices[0]
+        profit = 0
```

Comment:
profit is better than unclear variables such as output. But it's not perfect. Use max_profit.
Also, this function's name is bad. The best name is compute_max_profit. It's a good time to see "Readable code", then you'll understand what I mean.